llpc/translator/lib/SPIRV/libSPIRV/SPIRVOpCode.h

```diff
@@ -75,7 +75,7 @@ inline bool isBitwiseOpCode(Op OpCode) {
   return (unsigned)OpCode >= OpBitwiseOr && (unsigned)OpCode <= OpBitwiseAnd;
 }
 
-inline bool isBinaryShiftLogicalBitwiseOpCode(Op OpCode) {
+inline bool isBinaryShiftBitwiseOpCode(Op OpCode) {
```

Comment:
Removed "Logical" from this name, since it does not include `isLogicalOpCode` ops.